src/blocks/interfaces/serialized-block-head.ts

```diff
@@ -5,4 +5,5 @@ import { SerializedBlock } from './serialized-block';
 
 export interface SerializedBlockHead extends SerializedBlock {
   hash_rate: number;
+  reward: string;
```

Comment:
I am okay with the reward in the string since the unit is ore. But curious about how we will use this reward attribute in the charts?